### DISCUSSION TITLE
Server Esa for Eph data.

### DISCUSSION THREAD

**Discussion by Mauri740** [dl9rdz/rdz_ttgo_sonde#398] [Q&A]
Please, could you check if the server for Eph data is currently up or if it is down? I don't get Eph data for Rs92 good but only: RS92 RINEX eph state: Fetch error 
-Wifi mode: 3 
-two Rs92 in the Qrg list
Thank You

**Comment by hr-ru** [comment]
I just checked manually: Looks good, file seems to exist:
```
$ ftp gssc.esa.int
Connected to gssc.esa.int.
220 Wing FTP Server ready...
Name (gssc.esa.int:hans): anonymous
331 Password required for anonymous
Password:
230 User anonymous logged in.
Remote system type is UNIX.
Using binary mode to transfer files.
ftp> cd /gnss/data/daily/2023/318
250 CWD command successful. "/gnss/data/daily/2023/318" is current directory.
ftp> ls brdc318*
200 Port command successful
150 Opening data channel for directory list.
-rw-rw-r-- 1 5510 gssc_adm          57417  Nov 15 14:09 brdc3180.23n.gz
-rw-rw-r-- 1 5510 gssc_adm          84071  Nov 15 14:10 brdc3180.23g.gz
226 Transfer successful.
```

You should have this in your config:
`gssc.esa.int/gnss/data/daily/%1$04d/%2$03d/brdc%2$03d0.%3$02dn.gz`
see https://github.com/dl9rdz/rdz_ttgo_sonde/issues/368#issuecomment-1698628807

**Comment by Mauri740** [comment]
The server link is correct but the reply report is still: 
RS92 RINEX eph state: Fetch error
a couple of days ago everything was working fine

**Comment by Mauri740** [comment]
![Immagine 2023-11-15 161114](https://github.com/dl9rdz/rdz_ttgo_sonde/assets/139569966/9f917295-3dc2-4e7e-bf39-8e14f7ea330b)

**Comment by Mauri740** [comment]
after many attemps on the QRG list and reboots, I got Good Status again. I have no idea what happened, I didn't change any parameters. Thank you for your job!
